lib/templates/deploy.posh.node.template

```diff
@@ -0,0 +1,54 @@
+
+# Node Helpers
+# ------------
+
+function selectNodeVersion() {
+  if ($env:KUDU_SELECT_NODE_VERSION_CMD -ne $null) {
+    # The following are done only on Windows Azure Websites environment
+    & $env:KUDU_SELECT_NODE_VERSION_CMD "$DEPLOYMENT_SOURCE{SitePath}" "$DEPLOYMENT_TARGET" "$DEPLOYMENT_TEMP"
```

Comment:
so after testing this, I found out that `& $env:KUDU_SELECT_NODE_VERSION_CMD` doesn't work because the value of `$env:KUDU_SELECT_NODE_VERSION_CMD` is set by Kudu as `node "D:\...\Kudu.Services.Web\bin\scripts\selectNodeVersion"` and you can't run that with `&` in powershell. I'm not sure if there is a way to run a variable `$var` in powershell if 

```powershell
$var = "app.exe `"path to something`""
```